plugins/parquet/parquetembed.cpp

```diff
@@ -169,8 +169,8 @@ arrow::Status ParquetHelper::openWriteFile()
     {
         StringBuffer filename;
         StringBuffer path;
-        splitFilename(destination.c_str(), nullptr, &path, &filename, nullptr, false);
-        Owned<IDirectoryIterator> itr = createDirectoryIterator(path.str(), filename.append("*.parquet").str());
+        splitFilename(destination.c_str(), nullptr, &path, &filename, &filename, false);
+        Owned<IDirectoryIterator> itr = createDirectoryIterator(path.str(), filename.insert(filename.length() - 8, '*'), false, false); // We want to insert '*' before ".parquet"
```

Comment:
Is 'destination' already known to definitely end in ".parquet" ? If so, it was fine as it was before this change.
If in theory it could take another form, then can't assume this filename.length() - 8 ..